src/main/resources/application.yml

```diff
@@ -47,7 +47,7 @@ external.api.distance:
 
 paketomat:
   min-population : 10000
-  size-of-paketomat: 5
+  size-of-paketomat: 1
```

Comment:
I assume that the value is changed for testing purposes, please revert it.

Reply:
Done